src/site/content/en/blog/javascript-and-google-search-io-2019/index.md

```diff
@@ -0,0 +1,169 @@
+    The rendering strategy influences the performance and robustness of your web apps.
+  </figcaption>
+</figure> 
+
+The rendering strategy of your website influences performance. It also has an impact on how bots like Googlebot interact with it. Server-side rendering or pre-rendering can improve the performance for users and crawlers. Those strategies execute JavaScript on the server to generate the initial HTML content. The browser can start rendering the HTML as it arrives over the network, making the page load faster.  The [rendering session at I/O](https://www.youtube.com/watch?v=k-A2VfuUROg)  or [the blog post about rendering on the web](https://developers.google.com/web/updates/2019/02/rendering-on-the-web) shows how server-side rendering and hydration can improve the performance and user experience of web apps with more examples of how these strategies are implemented.
+
+If you are looking for a workaround to help crawlers that don't execute JavaScript or cannot make changes to your frontend codebase, consider [dynamic rendering](https://developers.google.com/search/docs/guides/dynamic-rendering) - we also have [a codelab that you can try out](https://codelabs.developers.google.com/codelabs/dynamic-rendering). Unlike server-side rendering or pre-rendering you won't get the user experience or performance benefits as dynamic rendering only serves static HTML to crawlers, so it's a workaround rather than a long-term strategy.
+
+## Test your pages
+
+While most pages generally work fine with Googlebot, we recommend testing your pages regularly to make sure your content is available to Googlebot and there are no problems. The easiest way to do a quick check of a page is the [Mobile-Friendly Test](https://g.co/mobilefriendly). Besides showing you issues with mobile-friendliness, it also gives you a screenshot of the above-the-fold content and the rendered HTML as Googlebot sees it.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="mobile-friendly-test-rendered-html.png" alt="The mobile-friendly test shows the rendered HTML Googlebot sees after rendering the page">
+  <figcaption class="w-figcaption">
+    The mobile-friendly test shows you the rendered HTML Googlebot uses.
+  </figcaption>
+</figure>
+
+You can also find out if there are resource loading issues or JavaScript errors.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="mobile-friendly-test-js-error.png" alt="The Mobile-Friendly Test shows JavaScript errors and a stack trace.">
+  <figcaption class="w-figcaption">
+    You can use the mobile-friendly test as well to find loading issues or JavaScript errors.
+  </figcaption>
+</figure>
+
+We recommend to verify your domain in [Google Search Console](https://g.co/searchconsole) so you can use the URL inspection tool to find out more about the crawling and indexing state of a URL, receive messages when we detect issues and find out more details of how your site performs in Google Search.
```

Comment:
```suggestion
[Google Search Console](https://g.co/searchconsole) is another great resource. Its URL inspection tool lets you learn about the crawling and indexing state of a URL, receive messages when the crawler detects issues, and get details about how your site performs in Google Search.
```
To avoid using first person, per the Documentation Style Guide.

Reply:
I rewrote your suggestion a bit to keep it a recommendation (which it is)